BO4E-dotnet/COM/Verbauchsmenge.cs

```diff
@@ -0,0 +1,15 @@
+using BO4E.ENUM;
+using BO4E.meta;
+using Newtonsoft.Json;
+
+namespace BO4E.COM
+{
+    /// <summary>Verbauchsmenge von Marktlokation</summary>
+    public class Verbauchsmenge : Menge
```

Comment:
Ich bin mir nicht sicher, ob wir das wirklich brauchen. Warum nehmen wir keine Menge und ergänzen den einen string in den UserProperties?